pyanaconda/modules/network/network.py

```diff
@@ -694,3 +700,6 @@ def log_configuration_state(self, msg_header, root_path=""):
 
     def _is_using_persistent_device_names(self, kernel_args):
         return 'net.ifnames.prefix' in kernel_args
+
+    def _is_using_biosdevname(self, kernel_args):
+        return kernel_args.get('biosdevname') == "1"
```

Comment:
Please, use `is_enabled` instead, that's meant for "anaconda style booleans".

Reply:
If I read the is_enabled correctly all these cases:
```
biosdevname
biosdevname=no
biosdevname=6
```
would make the biosdevname package installed on the target system. Maybe it is not big deal, but I'll rather ask @jstodola  what he thinks. Should we require exactly setting to value "1" or not ?

Reply:
See also https://bugzilla.redhat.com/show_bug.cgi?id=1623621#c2, I'll check the 71-biosdevname.rules file.

Reply:
In that case, the current implementation with `get` is correct.